### PR TITLE
cn: increase code coverage of handler.go

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- To increase the coverage of cn package, tests are added for handler.go
- Unnecessary `else` statements in handler.go are removed.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/cn/handler.go [hunk 1]
```diff
@@ -1323,8 +1320,6 @@ func (pm *ProtocolManager) GetPeers() []common.Address {
 				continue
 			}
 			addr = crypto.PubkeyToAddress(*pubKey)
```
Comment (KimKyungup) [inline node/cn/handler.go]: How about?
```suggestion
			addr = crypto.PubkeyToAddress(*pubKey)
			p.SetAddr(addr)
```

File: node/cn/handler.go [hunk 2]
```diff
@@ -1276,10 +1276,7 @@ func (pm *ProtocolManager) FindPeers(targets map[common.Address]bool) map[common
...
```
Comment (KimKyungup) [inline node/cn/handler.go]:
```suggestion
			addr = crypto.PubkeyToAddress(*pubKey)
			p.SetAddr(addr)
```

File: node/cn/handler_test.go [hunk 3]
```diff
@@ -17,19 +17,51 @@
...
+	"github.com/klaytn/klaytn/networks/p2p/discover"
 	"github.com/klaytn/klaytn/node"
 	"github.com/klaytn/klaytn/node/cn/mocks"
+	"github.com/klaytn/klaytn/params"
 	"github.com/stretchr/testify/assert"
 	"math/big"
 	"testing"
 )
 
 var blockNum1 = 20190902
```
Comment (jeonghonah) [inline node/cn/handler_test.go]:
```suggestion
const blockNum1 = 20190902
```

**Comment by ehnuje** [comment]
@jeonghonah @KimKyungup @nogang Could you please review this change?

File: node/cn/handler_test.go [hunk 4]
```diff
@@ -158,3 +191,320 @@ func TestBroadcastBlockHash(t *testing.T) {
...
+	defer mockCtrl.Finish()
+
+	peers := newPeerSet()
+	pm.peers = peers
+
+	cnPeer := NewMockPeer(mockCtrl)
+	pnPeer := NewMockPeer(mockCtrl)
+	enPeer := NewMockPeer(mockCtrl)
+
+	peers.cnpeers[addr1] = cnPeer
```
Comment (nogang) [inline node/cn/handler_test.go]: It seems that addr1,2,3 is empty because init () is not executed.

Comment (nogang) [reply node/cn/handler_test.go]: It's my mistake init() is always performed before this test code

**Pull request merged**